content/blog/2019-01-15-waterinfo-tidal-eel.md

```diff
@@ -0,0 +1,173 @@
+---
+slug: "waterinfo-tidal-eel"
+title: wateRinfo - Downloading tidal data to understand the behaviour of a migrating eel
+authors:
+  - Stijn Van Hoey
+  - Peter Desmet
+date: 2019-01-15
+categories: blog
+topicid:
+tags:
+  - software-peer-review
+  - r
+  - package
+  - water
+  - data-access
+  - tidal-data
+  - fishes
+  - movement-data
+  - acoustic-telemetry
+---
+
+> Do you know what that sound is, Highness? Those are the Shrieking Eels — if you don't believe me, just wait. They always grow louder when they're about to feed on human flesh. If you swim back now, I promise, no harm will come to you. I doubt you will get such an offer from the Eels.
+>
+> *Vizzini, [The Princess Bride](https://youtu.be/KGcat9tGZVU)*
+
+European eels ([*Anguilla anguilla*](https://en.wikipedia.org/wiki/Anguilla_anguilla)) have it tough. Not only are they depicted as monsters in movies, they are [critically endangered](https://doi.org/10.2305/IUCN.UK.2014-1.RLTS.T60344A45833138.en) in real life. One of the many aspects that is contributing to their decline is the reduced connectivity between their freshwater and marine habitats. Eels are catadromous: they live in freshwater, but migrate to the Sargasso Sea to spawn, a route that is blocked by numerous human structures (shipping locks, sluices, pumping stations, etc.). [Pieterjan Verhelst](http://www.marinebiology.ugent.be/node/68491) studies the impact of these structures on the behaviour of eels, making use of the [fish acoustic receiver network](http://lifewatch.be/en/fish-acoustic-receiver-network) that was established as part of the Belgian LifeWatch observatory. This animated video gives a quick introduction to his research and the receiver network:
+
+[![Tagging research on eel in Belgium](https://img.youtube.com/vi/7YQVgl3QPyY/0.jpg)](https://www.youtube.com/watch?v=7YQVgl3QPyY)
+
+In this blog post, we'll explore if the migration of one eel is influenced by the tide. It's a research use case for our R package [wateRinfo](https://ropensci.github.io/wateRinfo/), which was recently [peer reviewed](https://github.com/ropensci/software-review/issues/255) (thanks to reviewer [Laura DeCicco](https://github.com/ldecicco-USGS) and editor [Karthik Ram](https://github.com/karthik) for their constructive feedback!) and accepted as a community-contributed package to [rOpenSci](https://github.com/ropensci/wateRinfo).
```

Comment:
please put `wateRinfo` in backticks in the post

Reply:
wateRinfo -> `wateRinfo` done